package.json

```diff
@@ -18,6 +18,7 @@
     "async-retry": "^1.2.3",
     "axios": "^0.19.0",
     "p-limit": "^2.2.1",
```

Comment:
I'm assuming you can get rid of `p-limit` now since you're not using it anymore?

Reply:
I believe it's still being used in: `src/update/update.ts`